kie-wb-common-react/boxed-expression-editor/README.md

```diff
@@ -0,0 +1,8 @@
+#Boxed Expression Editor
+
+This editor provides the possibility to edit the expression related to a Decision Node, or to a Business Knowledge Model's function.
+
+##Scripts
+```sh
+WIP
```

Comment:
I believe we could add the scripts present `package.json` with a brief description. However, I don't think it's blocking for the merge :-)

Reply:
Totally agree. Since I am deeply changing that part with a showcase app that actually uses the boxed-expression-editor component, I will accomplish a more meaningful `README.md` in the next PR :-)